docs/tutorials/prowler-app-security-hub-integration.md

```diff
@@ -0,0 +1,244 @@
+    - **Archive Previous Findings:** Enable to automatically archive resolved findings
+    ![Integration settings](./img/security-hub/integration-settings.png)
+
+4. Configure authentication (optional):
+
+    - **Use Provider Credentials:** Uses the AWS provider's existing credentials (default)
+    - **Custom Credentials:** Configure separate credentials for Security Hub access
+
+5. For custom credentials:
+    - **Access Key ID & Secret:** AWS credentials with Security Hub permissions
+    - **SDK default credentials:** Uses the AWS SDK default credentials
+    - **IAM Role (Optional):** Role ARN and External ID for role assumption
+    ![Credentials configuration](./img/security-hub/credentials-configuration.png)
+
+6. Click **Create integration** to enable the integration and test the connection
+   ![Create integration](./img/security-hub/create-integration.png)
+
+???+ success
+    Once configured, findings from the next scan will appear in AWS Security Hub. Check the next scheduled scan time below the Security Hub connection status.
+
+### Finding Export Options
+
+**Send Only Failed Findings** reduces AWS Security Hub costs by filtering out PASS findings. This option is enabled by default and recommended for most use cases.
+
+**Archive Previous Findings** automatically archives findings that have been resolved since the last scan, maintaining clean Security Hub dashboards.
+
+---
+
+## Integration Management
+
+### Test Connection
+
+Use the **Test Connection** button to:
+
+- Verify AWS credentials validity
+- Check Security Hub accessibility
+- Detect enabled regions automatically
+- Validate finding export capability
+
+### Update Configuration
+
+Modify integration settings anytime:
+
+1. Click the **Config** button
+2. Update provider selection or finding filters
+3. Click **Update Configuration** to save
+
+### Enable/Disable Integration
+
+Toggle the integration status without deleting:
+
+- **Enable:** Resume finding exports to Security Hub
+- **Disable:** Pause exports temporarily
+
+### Delete Integration
+
+To remove the Security Hub integration:
+
+1. Click the **Delete** button in the integration row
+2. Confirm deletion in the modal dialog
+
+???+ warning
+    Deletion is permanent and cannot be undone. Consider using Enable/Disable for temporary changes.
+
+---
+
+## Viewing Findings in AWS Security Hub
+
+After configuration and scan completion, Prowler findings appear in AWS Security Hub:
+
+1. Open the [AWS Security Hub console](https://console.aws.amazon.com/securityhub/)
+2. Navigate to **Findings**
+3. Filter by **Product Name: Prowler** to view exported findings
+4. Click any finding title to see detailed information and compliance mappings
+
+???+ info "Detailed Instructions"
+    For comprehensive information on viewing and managing Prowler findings in Security Hub, refer to the [Viewing Prowler Findings guide](./aws/securityhub.md#viewing-prowler-findings-in-aws-security-hub).
```

Comment:
Again, I recomend you to directly link to the page where this is explained because we are going to unify docs as soon as possible. We want to avoid duplicities.